doc/Type/Real.pod6

```diff
@@ -68,11 +68,20 @@ it is an L<Int|/type/Int> when we instantiated C<Temperature> with an L<Int|/typ
 Converts the number to a C<Complex> with the number converted to a C<Num> as
 its real part and 0e0 as the imaginary part.
 
+=head2 method Int
+
+    method Int(Real:D:)
+
+Calls the L«C<Bridge> method|/routine/Bridge» on the invocant and then
+calls the L«C<Int> method|/routine/Int» on its return value.
```

Comment:
The problem with copy/paste is that it propagates (maybe minor) errors like this one. It would have been better to not repeat "call"

Reply:
Sure ! I will reformulate these. Thanks !